assets/js/main.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @author David O'Trakoun <me@davidosomething.com>
+ * @module global
+ * @requires module:shareIntent
+ */
+
+import analytics from './analytics.js';
+import { configs, bindSharePopup } from './shareIntent.js';
+
+bindSharePopup(configs.facebook);
+bindSharePopup(configs.google);
+bindSharePopup(configs.twitter);
+
+/**
+ * clickedBioProfileLink
+ *
+ * @param {Event} e
+ */
+const clickedBioProfileLink = (e) => {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).